content/millennium/r4/proprietary-codes.md

```diff
@@ -69,6 +69,15 @@ The Patient Resource supports proprietary codes for:
 * Patient.identifier.type codes are maintained in [Code Set 4 Person Identifier (Alias) Types](#code-set-4-person-identifier-alias-types)
 * Patient.contact.relationship codes for are maintained in [Code Set 351 Person Relationship Types](#code-set-351-person-relationship-types)
 
+
```

Comment:
remove the added blank lines before and after this

Reply:
remove blank line
https://github.com/cerner/fhir.cerner.com/pull/254/commits/6d1dc18c06362fc770aea2991e8e317ec263a54e#diff-f1a7812a3d6e9f4049fb6005eecfb1f5R77

Reply:
looks like the anchor needs to be changed back to 'relationships'

Reply:
https://github.com/cerner/fhir.cerner.com/pull/254/commits/c5efec01ed6e5832657292dcc035a08913ac1e49#diff-f1a7812a3d6e9f4049fb6005eecfb1f5R77

Reply:
also removed name of person
https://github.com/cerner/fhir.cerner.com/pull/254/commits/c5efec01ed6e5832657292dcc035a08913ac1e49#diff-e77269d0d7c7c9767d78a8e4f8e121a0R13
https://github.com/cerner/fhir.cerner.com/pull/254/commits/26b9f019f437539fcb31f31446bd3395662161f0#diff-e77269d0d7c7c9767d78a8e4f8e121a0R13